app/build.gradle

```diff
@@ -589,3 +592,5 @@ task checkstyle(type: Checkstyle) {
 afterEvaluate {
     check.dependsOn 'findbugs', 'pmd', 'checkstyle'
 }
+
+apply plugin: 'com.google.gms.google-services'
```

Comment:
Would you consider to move this `apply plugin ...` to the beginning of the `build.gradle`? It seems a conversion to have all the `apply plugin` there.